CHANGES.rst

```diff
@@ -26,6 +26,8 @@ Mosviz
 
 - Added coordinates display panels for Mosviz viewers. [#1795]
 
+- ``mosviz_row`` metadata now included in NIRISS-parsed 1D spectra. [#1836]
```

Comment:
Shouldn't this be in the bugfix subsection of 3.2?

Reply:
woops, good catch!  Or maybe it could be merged with the changelog entry of #1790 if the bug wasn't exposed until then?

Reply:
Re: merging -- I'll leave that to your good judgement.

Reply:
Hmm... Good points! Let me decide where to move it and I'll push again!